java/adapter/jdbc/src/test/java/org/apache/arrow/adapter/jdbc/Table.java

```diff
@@ -0,0 +1,235 @@
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.arrow.adapter.jdbc;
+
+import java.math.BigDecimal;
+import java.nio.charset.StandardCharsets;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * POJO to handle the YAML data from the test YAML file.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Table {
+    private String name;
+    private String type;
+    private String vector;
+    private String timezone;
+    private String create;
+    private String[] data;
+    private String query;
+    private String drop;
+    private String[] values;
+    private String[] vectors;
+    private int rowCount;
+
+    public Table() {
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getVector() {
+        return vector;
+    }
+
+    public void setVector(String vector) {
+        this.vector = vector;
+    }
+
+    public String[] getValues() {
+        return values;
+    }
+
+    public void setValues(String[] values) {
+        this.values = values;
+    }
+
+    public Long[] getLongValues() {
+        Long[] arr = new Long[values.length];
+        int i = 0;
+        for (String str: values) {
+            arr[i++] = Long.parseLong(str);
+        }
+        return arr;
+    }
+
+    public Integer[] getIntValues() {
+        Integer[] arr = new Integer[values.length];
+        int i = 0;
+        for (String str: values) {
+            arr[i++] = Integer.parseInt(str);
+        }
+        return arr;
+    }
+    
+    public Boolean[] getBoolValues() {
+        Boolean[] arr = new Boolean[values.length];
+        int i = 0;
+        for (String str: values) {
+            arr[i++] = Boolean.parseBoolean(str);
+        }
+        return arr;
+    }
+
+    public BigDecimal[] getBigDecimalValues() {
+        BigDecimal[] arr = new BigDecimal[values.length];
+        int i = 0;
+        for (String str: values) {
+            arr[i++] = new BigDecimal(str);
+        }
+        return arr;
+    }
+
+    public Double[] getDoubleValues() {
+        Double[] arr = new Double[values.length];
+        int i = 0;
+        for (String str: values) {
+            arr[i++] = Double.parseDouble(str);
```

Comment:
I don't think this indentation is correct and consistent with what is followed in rest of the codebase

Reply:
Can you please do checkstyle validation on all files?